Makefile

```diff
@@ -187,7 +187,12 @@ publish_mix: compile
 
 #==> Tests tasks
 
-test: test_erlang test_elixir
+test:
+	$(RM) -r .home
+	mkdir .home
+	HOME=$(CURDIR)/.home $(MAKE) test_erlang
```

Comment:
I have just realized this is not very effective. We can run tests directly with `make test_erlang` and the HOME directory won't be changed. Maybe we should provide a test task specific for distributions that sets the proper .home and whatever change we need in the future?